### ISSUE TITLE
A question about the performance against the official implementation

### ISSUE THREAD

**Issue by Vandermode** [Coldog2333/pytoflow#5]
Hi, thank you for sharing your nice reimplementation.

I was wondering whether you have any idea where the performance gap between this version and the authors' original implementation comes from?

Did you have any idea how to improve your implementation?
Thank you!

**Comment by Coldog2333** [comment]
Hello. I have no idea about the gap neither. I remember there is a hyperparameter (learning rate) I used is different from the original paper, but I forgot whether it was for the denoising task. Although hyperparameters matter sometimes but I don't think that's the main reason, I modified it just because I couldn't reproduce the performance with the same hyperparams. And I think that might be a written mistake.

No matter whether the learning rate does matter, you can check it in ./train.py and try to retrain it with the same hyperparams as the paper said, if necessary.

If you have other questions, feel free to contact me or open a new issue.
